addon/experiment-api/contactsApi.js

```diff
@@ -53,6 +100,61 @@ var convContacts = class extends ExtensionCommon.ExtensionAPI {
             args
           );
         },
+        async composeNew(properties) {
+          const window =
+            properties.windowId !== null
+              ? windowManager.get(properties.windowId, context).window
+              : Services.wm.getMostRecentWindow("mail:3pane");
+          const { to } = properties;
+          composeMessageTo(to, window.gFolderDisplay.displayedFolder);
```

Comment:
I think I'd like to make this more similar to the new APIs where we can pass the identity into this one, but I'll look at that after this lands.

Reply:
So you want `to` to be an array? https://thunderbird-webextensions.readthedocs.io/en/68/compose.html  (which totally bit me when I was working with the webextension API...It would be much nicer to accept an array or a single object.)

Reply:
Oh I was looking at the latest ones (select at the bottom of the left-hand frame), and mainly thinking about the identity, but don't worry, I'll take a look when I get time. I want to see if it is also viable to start adding the WebExtension based ids to contacts (I think that might need the latest beta though).